docs/reference/configuration.md

```diff
@@ -61,15 +61,15 @@ Global variable values may be any of the data types in the table below or lists
-| `no_diff`                 | Don't show diffs for changed models (Default: False)                                                                                                                                                                                                    |       boolean        |    N     |
-| `no_prompts`              | Disables interactive prompts in CLI (Default: True)                                                                                                                                                                                                    |       boolean        |    N     |
+| Option                    | Description                                                                                                                                                                                                                                             | Type                 | Required |
+|---------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------:|:--------:|
+| `auto_categorize_changes` | Indicates whether SQLMesh should attempt to automatically [categorize](../concepts/plans.md#change-categories) model changes during plan creation per each model source type ([additional details](../guides/configuration.md#auto-categorize-changes)) | dict[string, string] | N        |
+| `include_unmodified`      | Indicates whether to create views for all models in the target development environment or only for modified ones (Default: False)                                                                                                                       | boolean              | N        |
+| `auto_apply`              | Indicates whether to automatically apply a new plan after creation (Default: False)                                                                                                                                                                     | boolean              | N        |
+| `forward_only`            | Indicates whether the plan should be [forward-only](../concepts/plans.md#forward-only-plans) (Default: False)                                                                                                                                           | boolean              | N        |
+| `enable_preview`          | Indicates whether to enable [data preview](../concepts/plans.md#data-preview) for forward-only models when targeting a development environment (Default: False for dbt projects if the target engine doesn't support cloning, True otherwise)           | boolean              | N        |
```

Comment:
Default: True unless edge case.

Reply:
You sure? This feels very vague

Reply:
i meant the ordering not the actual content, put the true one first then describe the edge case

Reply:
aaaah, sorry, I misunderstood